java/java.source.base/src/org/netbeans/api/java/source/TreeMaker.java

```diff
@@ -267,6 +267,30 @@ public CaseTree Case(List<? extends ExpressionTree> patterns, Tree body) {
         return delegate.Case(patterns, body);
     }
     
+    /**
+     * Creates a new CaseTree for a rule case (case &lt;constants&gt; -> &lt;body&gt;).
+     *
+     * @param patterns the labels for this case statement.
+     * @param body the case's body
+     * @see com.sun.source.tree.CaseTree
+     * @since 2.39
```

Comment:
The `@since` should be updated, module's spec version incremented; and apichanges.xml filled.